packages/pages/src/common/src/function/internal/loader.test.ts

```diff
@@ -26,27 +28,35 @@ describe("loadTemplateModules", () => {
 
   it("loads and transpiles raw templates", async () => {
     const functionFile: path.ParsedPath[] = [
-      path.parse("tests/fixtures/src/functions/http/[param].ts"),
+      path.parse(path.join(httpPath, "[param].ts")),
     ];
 
     const functionModules = await loadFunctionModules(
       functionFile,
       true,
       projectStructure
     );
-    commonTests(functionModules, "param-47543");
+    if (path.sep === path.posix.sep) {
+      commonTests(functionModules, "param-47543");
+    } else {
```

Comment:
Why do we need an else case? The hash we add is the path up to and including the filename. I assume that's the issue. Instead, we should convertToPosixPath in the function code that generates the hash.

See the `unsecureHashPluginName(relativePath)` line in functionMetadataParser.ts